tests/python/visitor_def.py

```diff
@@ -0,0 +1,14 @@
+#ERROR: match
+def __eq__(self, other):
```

Comment:
@aryx can you also add a test for nested functions if you have time?

Reply:
there is already one 5 lines below